docs/tools/install.sh

```diff
@@ -0,0 +1,228 @@
+
+        if [ -z "${drive}" ]; then
+            echo "[ERROR] volume ${name}: empty drive name"
+            (( ++ecount ))
+            continue
+        fi
+
+        if [ -z "${driveNameFSUUIDMap[$drive]}" ]; then
+            echo "[ERROR] volume ${name}: FSUUID not found for drive ${drive}"
+            (( ++ecount ))
+            continue
+        fi
+    done
+
+    if [ "${ecount}" -gt 0 ]; then
+        exit 1
+    fi
+}
+
+function init() {
+    help_flag=0
+    for arg in "$@"; do
+        if [ "${arg}" != "apply" ]; then
+            directpv_args+=("${arg}")
+            if [ "${arg}" == "--help" ]; then
+                help_flag=1
+            fi
+        else
+            apply_flag=1
+        fi
+    done
+
+    if [ "${help_flag}" -eq 1 ]; then
+        cat << EOF
+USAGE:
+  install.sh [INSTALL-FLAGS] [apply]
+
+ARGUMENTS:
+  INSTALL-FLAGS    Optional DirectPV installation flags.
+  apply            If present, apply manifest file after generation.
+
+EXAMPLES:
+  # Generate DirectPV manifests.
+  $ install.sh
+
+  # Generate DirectPV manifests with private registry.
+  $ install.sh --registry private-registry.io --org org-name
+
+  # Generate DirectPV manifests with node-selector.
+  $ install.sh --node-selector node-label-key=node-label-value
+
+  # Generate and apply DirectPV manifests.
+  $ install.sh apply
+EOF
+        exit 255
+    fi
+
+    if ! which kubectl >/dev/null 2>&1; then
+        echo "kubectl not found; please install"
+        exit 255
+    fi
+
+    if kubectl directpv --version >/dev/null 2>&1; then
+        plugin_cmd=( kubectl directpv )
+    elif which kubectl-directpv >/dev/null 2>&1; then
+        plugin_cmd=( kubectl-directpv )
+    elif ./kubectl-directpv --version >/dev/null 2>&1; then
+        plugin_cmd=( ./kubectl-directpv )
+    else
+        echo "kubectl directpv plugin not found; please install"
+        exit 255
+    fi
+}
+
+function wait_for_crd() {
+    while ! kubectl get --ignore-not-found=true crd directpvdrives.directpv.min.io --no-headers -o NAME | grep -q .; do
+        echo "  ...waiting for drive CRD to be created"
+        sleep 1
+    done
+
+    while ! kubectl get --ignore-not-found=true crd directpvvolumes.directpv.min.io --no-headers -o NAME | grep -q .; do
+        echo "  ...waiting for volume CRD to be created"
+        sleep 1
+    done
+}
+
+function main() {
+    echo "* Probe legacy DirectCSI"
+    legacy=0
+    migrate=0
+    if is_legacy_drive_found || is_legacy_volume_found; then
+        legacy=1
+        migrate=1
+        check_directcsi_consistency
+    elif is_migrated_volume_found; then
+        legacy=1
+    fi
+
+    echo "* Generate DirectPV manifests to be saved at directpv.yaml"
+    if [[ "${legacy}" -eq 0 ]]; then
+        run_plugin_cmd install --declarative "${directpv_args[@]}" > directpv.yaml
+    else
+        run_plugin_cmd install --legacy --declarative "${directpv_args[@]}" > directpv.yaml
+    fi
+
+    if [ "${apply_flag}" -eq 1 ]; then
```

Comment:
```suggestion
    if [ "${apply_flag}" -eq 0 ]; then
```

Currently, it applies even if `apply` flag is not provided

Reply:
done